docs/NewArch.md

```diff
@@ -0,0 +1,74 @@
+
+## Creating a new Architecture Application
+
+1. Create a new React Native Application.
+
+```bat
+npx --yes @react-native-community/cli@latest init MyApp --version "latest"
+```
+
+2. Add React Native Windows as a dependency.
+
+```bat
+yarn add react-native-windows@latest
+```
+
+3. Add the New Architecture Template to your project. (Other templates can be found in [init-windows-cli](init-windows-cli.md#templates))
+
+```bat
+yarn react-native init-windows --template cpp-app --overwrite --logging
+```
+
+4. Run your App.
+
+```bat
+yarn react-native run-windows --logging
+```
+
+## Milestones
+
+Our development progress is organized into milestones, each with clear goals to guide our work. We are currently focused on achieving full API parity and enhancing accessibility. Note that **community modules are not yet supported** in this soft launch phase, so most (if not all) won’t work in new architecture applications at this stage.
+
+| Milestone | Milestone Nickname | Milestone Description | 
+| -- | -- | -- | 
+| ☑️ M0 | Proof of Concept | <li>Internal contributors can manually create a Win32 app that renders JSX using Composition</li> | 
+| ☑️ M1  | Experimental | <li>User can successfully initialize, build, and run a React Native Windows app on the new architecture. </li><li>User can use the _most common_ props/API’s.</li><li>User can use the _most common_ accessibility props/API’s. </li><li>User can observe _basic_ accessibility support within their app.</li><li>User will _not_ have access to full API parity with Paper.</li><li>User can use community modules within their app.</li><li>User _cannot_ use community modules with native UI within their app.</li> | 
+| 🔜 M2 | Parity and Accessibility | <ul><li>User can use all props/API’s that were supported on Paper. </li><li>User can use all accessibility props/API’s that were supported on Paper. </li><li>User can observe compliant/delightful accessibility support within their app.</li></ul> | 
+| ⬜ M3 | Ready for Modules | <li>User can use the _subset_ of community modules with native UI which have support for Fabric on Windows. </li> | 
+| ⬜ M4 | Production Ready **_(Fabric now officially in support)_** | <li>User can use most community modules with native UI which had been supported on Paper.</li><li>User can view documentation for the new architecture on the React Native Windows website. </li><li>Platform will be validated against _most common_ app scenarios. </li> | 
+| ⬜ M5 | New Default | <li>Platform will be validated against all app scenarios.</li><li>Paper architecture will be _deprecated_; Fabric will be the new default.</li> | 
```

Comment:
I think this is going to be in flux and I'd rather keep on the issue rather than in the docs.